Index: ovs/extensions/db/arakoon/arakoon/ArakoonManagement.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ovs/extensions/db/arakoon/arakoon/ArakoonManagement.py
+++ ovs/extensions/db/arakoon/arakoon/ArakoonManagement.py
@@ -26,8 +26,8 @@
 import string
 import logging

-from arakoon import Arakoon
-from arakoon.ArakoonExceptions import ArakoonNodeNotLocal
+import Arakoon
+from ArakoonExceptions import ArakoonNodeNotLocal

 def which_arakoon():
     path = '/'.join([X.appDir,"arakoon/bin/arakoon"])

--- ovs/extensions/db/arakoon/arakoon/Compat.py
+++ ovs/extensions/db/arakoon/arakoon/Compat.py
@@ -24,7 +24,6 @@
 else:
     logging.info("opting for normal client")
-    print "arakoon"
-    from arakoon import Arakoon
+    import Arakoon
     arakoon_client = Arakoon


@@ -213,5 +213,4 @@

 def which_compat():
-    print "which_compat"
     g = globals()
     if sys.prefix == '/opt/qbase3':
         r = Q()
