--- ovs/extensions/db/arakoon/arakoon/ArakoonProtocol.py
+++ ovs/extensions/db/arakoon/arakoon/ArakoonProtocol.py
@@ -26,7 +26,6 @@
 import os.path
 import ssl
 import struct
-import logging
 import select
 import operator
 import cStringIO
@@ -38,6 +37,9 @@
 ARA_CFG_CONN_TIMEOUT = 60
 ARA_CFG_CONN_BACKOFF = 5
 ARA_CFG_NO_MASTER_RETRY = 60
+
+from ovs.log.logHandler import LogHandler
+logging = LogHandler.get('arakoon', name='internal')

 class ArakoonClientConfig :

--- ovs/extensions/db/arakoon/arakoon/Compat.py
+++ ovs/extensions/db/arakoon/arakoon/Compat.py
@@ -1,11 +1,14 @@
 import os
 import ConfigParser
 import shutil
-import logging
 import subprocess
 import StringIO
 import fnmatch
 import sys
+from ovs.log.logHandler import LogHandler
+
+logging = LogHandler.get('arakoon', name='internal')
+

 class Status:
     HALTED = 'HALTED'
@@ -24,6 +24,5 @@
 else:
     logging.info("opting for normal client")
-    print "arakoon"
-    from arakoon import Arakoon
+    import Arakoon
     arakoon_client = Arakoon

@@ -213,5 +213,4 @@

 def which_compat():
-    print "which_compat"
     g = globals()
     if sys.prefix == '/opt/qbase3':
         r = Q()
--- ovs/extensions/db/arakoon/arakoon/RemoteControlProtocol.py
+++ ovs/extensions/db/arakoon/arakoon/RemoteControlProtocol.py
@@ -15,7 +15,6 @@
 """

 import socket
-import logging
 import struct


--- ovs/extensions/db/arakoon/arakoon/Nursery.py
+++ ovs/extensions/db/arakoon/arakoon/Nursery.py
@@ -24,4 +24,5 @@
-import logging
+from ovs.log.logHandler import LogHandler
+logging = LogHandler.get('arakoon', name='internal')

 maxDuration = 3

--- ovs/extensions/db/arakoon/arakoon/ArakoonRemoteControl.py
+++ ovs/extensions/db/arakoon/arakoon/ArakoonRemoteControl.py
@@ -15,8 +15,9 @@
 """

 import socket
-import logging

+from ovs.log.logHandler import LogHandler
+logging = LogHandler.get('arakoon', name='internal')
 import RemoteControlProtocol as RCP

 def collapse(ip, port, clusterId, n):
--- ovs/extensions/db/arakoon/arakoon/NurseryRouting.py
+++ ovs/extensions/db/arakoon/arakoon/NurseryRouting.py
@@ -15,6 +15,4 @@
 """


-import logging
-
 class RoutingInfo:
--- ovs/extensions/db/arakoon/arakoon/ArakoonManagement.py
+++ ovs/extensions/db/arakoon/arakoon/ArakoonManagement.py
@@ -26,8 +26,9 @@
 import string
-import logging
+from ovs.log.logHandler import LogHandler
+logging = LogHandler.get('arakoon', name='internal')

-from arakoon import Arakoon
-from arakoon.ArakoonExceptions import ArakoonNodeNotLocal
+import Arakoon
+from ArakoonExceptions import ArakoonNodeNotLocal

 def which_arakoon():
     path = '/'.join([X.appDir,"arakoon/bin/arakoon"])

--- ovs/extensions/db/arakoon/arakoon/ArakoonValidators.py
+++ ovs/extensions/db/arakoon/arakoon/ArakoonValidators.py
@@ -20,6 +20,5 @@
 from ArakoonExceptions import ArakoonInvalidArguments

 import ArakoonProtocol
-import logging
 from functools import wraps
